doc/classes/ARVRAnchor.xml

```diff
@@ -60,6 +60,7 @@
 	</methods>
 	<members>
 		<member name="anchor_id" type="int" setter="set_anchor_id" getter="get_anchor_id">
+			The anchor's id. You can set this before the anchor itself exists. The first anchor that gets an id of [code]1[/code], the second an id of [code]2[/code], etc. When anchors get removed the corresponding id stays free. The engine can then assign it to new anchors. The most common situation where anchors 'disappear' is when the AR server identifies that two anchors represent different parts of the same plane and merge them.
```

Comment:
- "The first anchor gets an id"
- merge - merges
- These 2 sentences could be combined:
> When anchors get removed the corresponding id stays free. The engine can then assign it to new anchors. 

When anchors are removed the corresponding id remains free and the engine can assign it to new anchors.